api/tests/Feature/UserTest.php

```diff
@@ -1031,4 +1037,421 @@ public function testFilterByJobLookingStatus(): void
+        User::factory()->count(5)->create([
+            'current_province' => null,
+            'location_preferences' => null,
+            'looking_for_english' => null,
+            'looking_for_french' => null,
+            'looking_for_bilingual' => null,
+        ]);
+
+        // Create some partially complete users.
+        User::factory()->count(2)->create([
+            'current_province' => 'ONTARIO',
+            'location_preferences' => null,
+            'looking_for_english' => false,
+            'looking_for_french' => true,
+            'looking_for_bilingual' => false,
+        ]);
+        User::factory()->count(2)->create([
+            'current_province' => 'ONTARIO',
+            'location_preferences' => ['PRAIRIE'],
+            'looking_for_english' => null,
+            'looking_for_french' => null,
+            'looking_for_bilingual' => null,
+        ]);
+
+        // Create some complete users.
+        User::factory()->count(3)->create([
+            'current_province' => 'ONTARIO',
+            'location_preferences' => ['PRAIRIE'],
+            'looking_for_english' => null,
+            'looking_for_french' => true,
+            'looking_for_bilingual' => null,
+            'telephone' => '+15407608748',
+            'current_city' => 'Somewhere random',
+            'is_gov_employee' => false,
+            'expected_salary' => ['_50_59K'],
+            'would_accept_temporary' => false,
+        ]);
+
+        // Assert query no isProfileComplete filter will return all users
+        $this->graphQL(/** @lang Graphql */ '
+            query getUsersPaginated($where: UserFilterAndOrderInput) {
+                usersPaginated(where: $where) {
+                    paginatorInfo {
+                        total
+                    }
+                }
+            }
+        ', [
+            'where' => []
+        ])->assertJson([
+            'data' => [
+                'usersPaginated' => [
+                    'paginatorInfo' => [
+                        'total' => 13
+                    ]
+                ]
+            ]
+        ]);
+
+        // Assert query with isProfileComplete filter set to true will return correct user count
+        $this->graphQL(/** @lang Graphql */ '
+            query getUsersPaginated($where: UserFilterAndOrderInput) {
+                usersPaginated(where: $where) {
+                    paginatorInfo {
+                        total
+                    }
+                }
+            }
+        ', [
+            'where' => [
+                'isProfileComplete' => true,
+            ]
+        ])->assertJson([
+            'data' => [
+                'usersPaginated' => [
+                    'paginatorInfo' => [
+                        'total' => 3
+                    ]
+                ]
+            ]
+        ]);
+
+        // Assert query with isProfileComplete filter set to false will return all users
```

Comment:
The other booleans also work this way since that's how they work in the candidate count query and I mostly just ported them over. Do you think I should change them as well?

Reply:
Oh, you're right!  I feel like I've had this discussion before. deja-vu...
No, don't change it.  Consistency is much more important.  See, the test is good for showing expected behaviour, too.